packages/nestjs-typeorm-history/src/common.ts

```diff
@@ -0,0 +1,42 @@
+import { Column } from 'typeorm';
+import { EntityManager, EntitySubscriberInterface } from 'typeorm';
+
+export interface History {
+  id: number | string;
+  entityId: number | string;
```

Comment:
version and dates. should we do that optional?

Reply:
See comments above